pytest_factoryboy/plugin.py

```diff
@@ -58,4 +58,4 @@ def pytest_runtest_call(item):
 def pytest_addhooks(pluginmanager):
     """Register plugin hooks."""
     from pytest_factoryboy import hooks
-    pluginmanager.addhooks(hooks)
+    pluginmanager.add_hookspecs(hooks)
```

Comment:
this will need a change to requirements pytest >= 2.8.0
and that version has a bug with xdist https://github.com/pytest-dev/pytest/issues/1062